client/components/Overview/components/ImageGallery/ImageGalleryImageSlide.jsx

```diff
@@ -12,24 +12,6 @@ const ImageGalleryImageSlide = ({
   thumbSplitArr, thumbDisplayArr, setThumbDisplayArr, setOverviewModal, LogClick,
 }) => (
   <div id="imagegallery-mainimage-container">
```

Comment:
What happened here

Reply:
the line above 11?

Reply:
There were so many components in the container and then it's all gone and I was curious! But ultimately approved!